assessments-service/src/main/java/com/transformuk/hee/tis/assessment/service/service/impl/AssessmentServiceImpl.java

```diff
@@ -163,13 +163,16 @@ public AssessmentDTO findOne(Long id) {
       Collection<Object> set = Collections
           .unmodifiableSet(permissionService.getUsersProgrammeIds());
       Specifications inProgrammes = Specifications
-          .where(SpecificationFactory.in("programmeId", set));
+          .where(in("programmeId", set));
       specs.add(inProgrammes);
     }
 
     // add the column filters criteria
     if (columnFilters != null && !columnFilters.isEmpty()) {
-      columnFilters.forEach(cf -> specs.add(in(cf.getName(), cf.getValues())));
+      columnFilters.forEach(cf -> {
+        List<Object> valuesList = SpecificationFactory.getDateAwareValuesFromColumnFilter(cf);
+        specs.add(in(cf.getName(), valuesList));
```

Comment:
`valuesList` can be included in the line below.

Reply:
Cleaner - thanks.